ocw-cli/cli_app.py

```diff
@@ -35,7 +36,10 @@
 from ocw.data_source.local import load_file
 import ocw.utils as utils
 import ocw.data_source.esgf as esgf
-from ocw_config_runner.configuration_writer import export_evaluation_to_config
+from example_package import *
+
+#from ocw_config_runner.configuration_writer import export_evaluation_to_config
+from configuration_writer import export_evaluation_to_config
```

Comment:
This one should be corrected with another JIRA ticket. Thank you for finding my mistake.